test/test_images/probe_helper/probe_helper_test.go

```diff
@@ -265,14 +265,30 @@ func runTestCloudSchedulerSource(ctx context.Context, group *errgroup.Group, per
 	})
 }
 
+type probeEventOption func(*cloudevents.Event)
+
+func withProbeSubject(subject string) probeEventOption {
+	return func(event *cloudevents.Event) {
+		event.SetSubject(subject)
+	}
+}
+
+func withProbeTimeout(timeout string) probeEventOption {
```

Comment:
Can we pass a Duration instead of a string?

Reply:
Done.